SharedPasswords/Credential.swift

```diff
@@ -7,6 +7,8 @@
 
 import Foundation
 
+public let sharedCredentialsKey = "sharedCredentials"
```

Comment:
I don't really like having this live at the top level. I'd rather have it live inside of something. Typically, I like to have a `Keys` struct that has `static let`s inside it with each of the needed keys. I'd be fine with it living inside of `Credential` since the object that used to contain it is gone now, and there is just the one.